.eleventy.js

```diff
@@ -1,12 +1,16 @@
 "use strict";
 
 const { addPlugins, addFilters, addTransforms } = require("./_11ty");
+const yaml = require("js-yaml");
 
 module.exports = function(eleventyConfig) {
     addPlugins(eleventyConfig);
     addFilters(eleventyConfig);
     addTransforms(eleventyConfig);
 
+    // Add YAML files as data sources
+    eleventyConfig.addDataExtension("yml", contents => yaml.safeLoad(contents));
```

Comment:
```suggestion
    eleventyConfig.addDataExtension("yml", yaml.safeLoad);
```

Reply:
I think we should leave it as-is for now since this is more explicit and a bit safer against either of these APIs changing.

FWIW, the examples are all written this way: https://www.11ty.dev/docs/data-custom/